requirements.txt

```diff
@@ -0,0 +1,12 @@
+requests>=2.5.1
+httpretty>= 0.8.3
+oauthlib>=0.6.3
+pyflakes>=0.8.1
+pytest>=2.6.3
+pytest-cache>=1.0
+pytest-cov>=1.8.0
+pytest-flakes>=0.2
+pytest-pep8>=1.0.6
+semantic_version>=2.3.1
+sphinx>=1.2.3
+
```

Comment:
updated all dependencies to latest version.  Removed Sphinx, oauthlib.  
What does 
```
--index-url https://pypi.python.org/simple/  
-e .
``` 
do?